ios/RCTMapboxGL/RCTMapboxGL.m

```diff
@@ -422,7 +438,10 @@ - (void)setVisibleCoordinateBounds:(MGLCoordinateBounds)bounds edgePadding:(UIEd
 - (void)selectAnnotation:(NSString*)selectedId animated:(BOOL)animated;
 {
     RCTMGLAnnotation * annotation = [_annotations objectForKey:selectedId];
-    if (!annotation) { return; }
+    if (!annotation) {
+        _selectedAnnotation=nil;
+        return; }
```

Comment:
Indentation is off here. Also, there should be spaces between operators and operands (before and after `=`)